test/common/stats/tag_extractor_impl_test.cc

```diff
@@ -166,16 +190,28 @@ TEST(TagExtractorTest, DefaultTagExtractors) {
   regex_tester.testRegex("cluster.ratelimit.ssl.ciphers.ECDHE-RSA-AES128-GCM-SHA256",
                          "cluster.ssl.ciphers", {cluster_tag, cipher_suite});
 
+  // SSL curve
+  regex_tester.testRegex("cluster.ratelimit.ssl.curves.X25519", "cluster.ssl.curves",
+                         {cluster_tag, ssl_curve});
+
+  // SSL signature algorithm
+  regex_tester.testRegex("cluster.ratelimit.ssl.sigalgs.rsa_pss_rsae_sha256", "cluster.ssl.sigalgs",
+                         {cluster_tag, ssl_sig_alg});
+
+  // SSL protocol version
+  regex_tester.testRegex("cluster.ratelimit.ssl.versions.TLSv1.2", "cluster.ssl.versions",
+                         {cluster_tag, ssl_version});
+
   // ipv6 non-loopback (for alphabetical chars)
   listener_address.value_ = "[2001_0db8_85a3_0000_0000_8a2e_0370_7334]_3543";
-  regex_tester.testRegex(
```

Comment:
same with this one. this old one no longer makes sense?

Reply:
I don't see metric `ssl.cipher.AES256-SHA` generated by any of Envoy v1.10.0, v1.11.0, v1.12.0, v1.13.0